src/App.js

```diff
@@ -1,25 +1,55 @@
-import logo from './logo.svg';
-import './App.css';
+/* Modules */
+import { Routes, Route } from 'react-router-dom';
+/* Components */
+import HomePage from './components/HomePage';
+import ContentPage from './components/ContentPage';
+import Footer from './components/Footer';
+/* Assets/Styling */
+import './App.scss';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+      <main>
+        <Routes>
+          <Route path="/" element={ <HomePage /> }/>
+          <Route path="/contentPage" element={ 
+          <ContentPage/> }/>
+        </Routes>
+        <Footer />
+      </main>
   );
 }
 
 export default App;
+
+
+// Pseudo Code 
+// On page load the user lands to a page where there are:
+		//Input form for plant name 
+			// Get and store the 'plantName' data in a STATE that will be connected to Firebase. 
+      // Make API call to Firebase - useEffect - axios? 
+			//Render the name on the page as a heading to the visual tracking chart. 
+
+// On submit - the user will be taken to a second input section 
+		// Goal: watering frequency per week 
+    // When user selects the option 
+      // Get and store the "value" for "watering frequency" data in a STATE that will be connected to Firebase. 
+      // Make API call to Firebase - useEffect - axios? 
+
+    //on click to "completed button" - conditionally render the visual goal tracker.  How to give the button different role based on the input? 
+			// This information will be stored in Firebase realtime database 
+			// The value of the selection will be rendered on the page through the    real-time database received from Firebase. 
+
+// Each time the user completes the goal - they click on the button underneath and it adds a visual cue that they completed one checkmark among however many they set for that week.
+		 // the updated information will be stored in Firebase. 
+
+// By the end of the week, the user can press 'How did I do?' button which  will turn conditionally render the color of the visual chart for green or red based on if they reached their goal or not. ../
+
+
+
+
+
+
```

Comment:
Overall, very neat idea and I like it a lot! great job! adding a unique user login to track certain users plants and stuff would probably be a great addition down the line! also the time stamping of each individual plant, AND adding maybe a weekly reset to all of the plants so that it resets... OR maybe just have it reset every Monday or on a specific day? great job!